### ISSUE TITLE
Documentation is out of date.

### ISSUE THREAD

**Issue by dconboy** [MicrosoftDocs/CloudAppSecurityDocs#372]
Other issues were raised about documentation and closed 6 months ago. I am currently following documentation as it is now, and noticed that several instructions/images do not correspond with what is in GSuite now.

**Comment by dcurwin** [comment]
Thank you for your comment. We will investigate the issue and get back to you.

**Comment by dcurwin** [comment]
Can you let us know which pages you find out of date?

**Comment by dconboy** [comment]
Step 7a, the screenshot is redundant as there is no area to choose user type (I found that I could only choose between internal/external later in the process.) Also the OAuth Consent Screen is different from what is displayed.

Step 9D is also changed on GSuite's end (Keys is in a new tab. I believe there are a few more inconsistencies that I can't see since I'm not viewing the steps via a new project and just viewing the one I created for GSuite to CAS.

GSuite seems to have moved stuff around and it took a bit more time trying to follow/guess next steps since the steps/images didn't line up with what GSuite displayed.
